docs/modules.md

```diff
@@ -282,6 +282,47 @@ the request is a server admin.
 Modules can modify the `request_content` (by e.g. adding events to its `initial_state`),
 or deny the room's creation by raising a `module_api.errors.SynapseError`.
 
+#### Presence router callbacks
+
+Presence router callbacks allow module developers to specify additional users (local or remote)
+to receive certain presence updates from local users. Presence router callbacks can be 
+registered using the module API's `register_presence_router_callbacks` method.
+
+The available presence router callbacks are:
+
+```python 
+async def get_users_for_states(
+    self,
+    state_updates: Iterable[synapse.api.UserPresenceState],
+) -> Dict[str, Set[synapse.api.UserPresenceState]]:
```

Comment:
Again, please quote Synapse's types:

```suggestion
    state_updates: Iterable["synapse.api.UserPresenceState"],
) -> Dict[str, Set["synapse.api.UserPresenceState"]]:
```

(quoting a type hint is standard in Python, for types that aren't imported in the file, or are imported conditionally - see type hints for `HomeServer` throughout the code pase)

Reply:
done